test/partiql-tests-runner/src/test/resources/config/eval/skip-eval.txt

```diff
@@ -18,6 +18,22 @@ STRICT:::divDecimalInt
 PERMISSIVE:::subtractionOutOfAllowedPrecision
 STRICT:::subtractionOutOfAllowedPrecision
 
+// TODO: Datetime methods for EXTRACT for TIME WITH TIMEZONE and TIMESTAMP WITH TIMEZONE
```

Comment:
Are these the conformance tests that were failing w/ this change?

Reply:
Yes, some issue introduced with lowering the Ion timestamp.